tubular/__init__.py

```diff
@@ -9,4 +12,6 @@
     numeric,
     strings,
 )
-from tubular._version import __version__
+
+with suppress(ModuleNotFoundError):
```

Comment:
P: took me a second to grok what happening here. But I like it more than my clunky suggestion that I made in the PR.